great_expectations/profile/user_configurable_profiler.py

```diff
@@ -118,12 +120,22 @@ def __init__(
                 MetricConfiguration("table.columns", {})
             )
         elif isinstance(self.profile_dataset, Validator):
+            context = self.profile_dataset.data_context
             self.all_table_columns = self.profile_dataset.get_metric(
                 MetricConfiguration("table.columns", {})
             )
         else:
             self.all_table_columns = self.profile_dataset.get_table_columns()
 
+        # Check to see if the user has disabled progress bars
+        self._enable_progress_bars = True
+        if context:
+            progress_bars = context.progress_bars
+            if progress_bars.get("globally") is False:
+                self._enable_progress_bars = False
+            elif progress_bars.get("profilers") is False:
+                self._enable_progress_bars = False
+
```

Comment:
I would expect that `progress_bars.get("profilers")` would override `progress_bars.get("globally")` irrespective of whether it is True or False. So something like:
```suggestion
            progress_bars = context.progress_bars
            self._enable_progress_bars = progress_bars.get("globally")
            
            if progress_bars.get("profilers") is not None:
                self._enable_progress_bars = progress_bars.get("profilers")

```

Reply:
ah yes that's what we had discussed! I knew I was missing something 🙇🏽